app/build.gradle

```diff
@@ -634,6 +626,25 @@ if (gradle.hasProperty('localProperties.dependencySubstitutions.geckoviewTopsrcd
     apply from: "${topsrcdir}/substitute-local-geckoview.gradle"
 }
 
+androidComponents {
+    onVariants(selector().all(), { variant ->
+        switch (variant.name) {
+            case "hvr3dofArm64CnDebug":
+                variant.applicationId = "cn.igalia.wolvic.tdof." + getDevApplicationIdSuffix()
```

Comment:
This fails to compile if `getDevApplicationIdSuffix()` returns an empty string.

Reply:
Ah because of the leading dot. Good catch